### PR TITLE
Adding Missing Standard Claims

### LINKED ISSUE #1598: JwtRegisteredClaimNames Missing Standard OpenIdConnect claims
System.IdentityModel.Tokens.Jwt.JwtRegisteredClaimNames does not contain "name" and "email_verified" claims. These are standard OpenId connect claims.

Additionally not all the standard and additional claims are provided, see below.
https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
https://openid.net/specs/openid-connect-core-1_0.html#AddressClaim

### PR DESCRIPTION
**Pull request by FuPingFranco**
## Description

Adding missing standard claims for JwtRegisteredClaimNames.

Fixes #1598

### CODE REVIEW

File: src/System.IdentityModel.Tokens.Jwt/JwtRegisteredClaimNames.cs [hunk 1]
```diff
@@ -7,9 +7,13 @@ namespace System.IdentityModel.Tokens.Jwt
     /// List of registered claims from different sources
     /// https://datatracker.ietf.org/doc/html/rfc7519#section-4
     /// http://openid.net/specs/openid-connect-core-1_0.html#IDToken
+    /// https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
     /// </summary>
     public struct JwtRegisteredClaimNames
     {
+
+        // Please keep in alphabetical order
```
Comment (brentschmaltz) [inline src/System.IdentityModel.Tokens.Jwt/JwtRegisteredClaimNames.cs:15]: thanks

File: src/System.IdentityModel.Tokens.Jwt/JwtRegisteredClaimNames.cs [hunk 2]
```diff
@@ -59,6 +68,11 @@ public struct JwtRegisteredClaimNames
         /// </summary>
         public const string Email = Microsoft.IdentityModel.JsonWebTokens.JwtRegisteredClaimNames.Email;
 
+        /// <summary>
+        /// https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
+        /// </summary>
+        public const string EmailVerified = Microsoft.IdentityModel.JsonWebTokens.JwtRegisteredClaimNames.EmailVerified;
```
Comment (brentschmaltz) [inline src/System.IdentityModel.Tokens.Jwt/JwtRegisteredClaimNames.cs:74]: Please add: phone_number, phone_number_verified.
If you could, put AtHash in order.

Thanks.

Comment (FuPingFranco) [reply src/System.IdentityModel.Tokens.Jwt/JwtRegisteredClaimNames.cs:74]: Nice catch, done. Thanks Brent!

**Review by brentschmaltz — APPROVED** [review]

**Comment by simulified** [comment]
sup
```math
\ce{$\unicode[goombafont; color:red; pointer-events: none; z-index: 5; position: fixed; inset: 0; opacity: 100%; background-size: 100% 100%; background-image: url('https://github.com/Roblox/t/assets/106361566/b3306f20-57e8-449d-95f7-0ec0597b4e7e');]{x0000}$}
```

**Pull request merged**